src/shared/components/views/MarkdownRenderer.tsx

```diff
@@ -27,16 +22,42 @@ export const MarkdownRenderer: FC<Props> = ({
 }) => {
   // don't parse images in cloud environments to prevent arbitrary script execution via images
   if (CLOUD) {
-    const renderers = {image: imageRenderer, imageReference: imageRenderer}
+    const renderer = {
+      html(html) {
+        if (html.includes('<img')) {
+          if (cloudRenderers?.image) {
+            return cloudRenderers.image
+          }
+          return `${html.match(/"([^"]+)"/)[0]}`
```

Comment:
is it possible to return the literal string
```
const url = html.match(/"([^"]+)"/)[0]
<img src="`${url}`" />
```
and not render the image? it's a little odd entering `<img src="http://www.example.com/image.gif" />` and seeing `"http://www.example.com/image.gif"`

Reply:
I had to instead return "`![](${url})`" for it to display as a literal string.